src/electron/build_main.action.mjs

```diff
@@ -26,13 +27,16 @@ const ELECTRON_BUILD_DIR = 'build';
 
 export async function main(...parameters) {
   const {platform, buildMode, networkStack, sentryDsn} = getBuildParameters(parameters);
+  const buildEnv = await getBuildEnvironment(platform, buildMode, sentryDsn);
 
   await runAction('www/build', platform, `--buildMode=${buildMode}`);
 
-  await webpackPromise({
-    ...electronMainWebpackConfig({networkStack}),
-    mode: getWebpackBuildMode(buildMode),
-  });
+  await webpackPromise(
+    electronMainWebpackConfigs({networkStack, buildEnv}).map(cfg => ({
```

Comment:
Let's pass the SENTRY_DNS and the app version explicitly so it's clear what we are using.